examples/nat_topo.py

```diff
@@ -125,10 +125,12 @@ def test():
 
     # If we couldn't show the Pax output in a separate window, show it now.
     if not config.X_windows:
+        sendInt(net, nat0)
         print "Show Pax output? (Y/n)"
         if (sys.stdin.read(1).upper() == "Y"):
```

Comment:
"Y/n" usually means that "Y" is the default case (that is, you get the positive branch taken even if you simply press Return). The code however runs the negative branch as the default case. This could be fixed by changing "Y/n" to "y/N", and replacing this line with `if (sys.stdin.read(1).lower() == "y"):`.